ReactiveObjC/RACTuple.h

```diff
@@ -111,15 +115,84 @@ NS_ASSUME_NONNULL_BEGIN
 - (nullable id)objectAtIndexedSubscript:(NSUInteger)idx;
 @end
 
+/// A tuple with exactly two generic values.
+@interface RACTwoTuple<__covariant First, __covariant Second> : RACTuple
```

Comment:
(I hope you don't consider it insolent to comment here as an outsider.)

Should there not be a `RACOneTuple` type as well? I can't think of a practical use case at the moment, but it seems to me that it may be useful for _something_, perhaps when bridging with Swift, or if nothing else then on the grounds of symmetry.

Related question: does it make sense now to have the nil tuple be its own type (like all the other tuples with a specified number of values), e.g. `RACZeroTuple`? That also seems like it'd be consistent with this new scheme.